service/src/main/java/com/epam/esm/service/impl/TagServiceImpl.java

```diff
@@ -26,30 +26,34 @@
 @Service
 @RequiredArgsConstructor
 public class TagServiceImpl implements TagService {
-    private final TagDao tagDao;
-    private final GiftCertificateDao giftCertificateDao;
+    private final DataTagDao tagDao;
+    private final DataGiftCertificateDao giftCertificateDao;
     private final ServiceTagMapper tagMapper;
-    private final ServicePageMapper pageMapper;
 
 
     @Transactional
     @Override
     public TagDto create(TagCreateDto tagCreateDto) {
         isTagDublicate(tagCreateDto);
         Tag tag = tagMapper.toEntity(tagCreateDto);
-        Tag insertedTag = tagDao.create(tag);
+        Tag insertedTag = tagDao.save(tag);
         return tagMapper.toDto(insertedTag);
     }
 
     @Override
     public List<TagDto> findAll(PageDto pageDto) {
-        Page page = pageMapper.toEntity(pageDto);
-        List<Tag> foundTags = tagDao.findAll(page);
+        Pageable pageRequest = PageRequest.of(pageDto.getNumber(), pageDto.getSize());
+        List<Tag> foundTags = tagDao.findAll(pageRequest).getContent();
         return foundTags.stream()
                 .map(tagMapper::toDto)
                 .collect(Collectors.toList());
     }
 
+    @Override
+    public TagDto create(TagDto value) {
+        return null;
```

Comment:
Please refactor this method: the motivation of returning `null` is unclear here.